lib/page-object/accessors.rb

```diff
@@ -1304,6 +1304,55 @@ def indexed_property (name, identifier_list)
     end
 
     #
+    # adds a method to return a specified class wrapping a generic Element object
+    #
+    # @example
+    #   page_section(:navigation_bar, :id => 'nav-bar')
+    #   # will generate 'navigation_bar' and 'navigation_bar?'
+    #
+    # @param [Symbol] the name used for the generated methods
+    # @param [Hash] identifier how we find an element.  You can use multiple parameters
+    #   by combining of any of the following except xpath.  The valid keys are:
+    #   * :class => Watir and Selenium
+    #   * :css => Selenium only
+    #   * :id => Watir and Selenium
+    #   * :index => Watir and Selenium
+    #   * :name => Watir and Selenium
+    #   * :xpath => Watir and Selenium
+    #
+    def page_section (name, section_class, identifier_list)
+      define_method("#{name}") do
```

Comment:
Since grabbing the entire text of a section isn't as valuable, and calling out that it is an element is slightly inaccurate, I used name for the method name. I feel like this leads to more succinct code since a section wraps a number of elements that have a particular value together, that value becomes the name.